README.md

```diff
@@ -95,6 +95,8 @@ If you're familiar with other popular [awesome lists](https://github.com/sindres
 
 ## Programming in General
 
+<a href="#contents"><img align="right" width="15" height="15" src="https://bit.ly/382Vmvi" alt="Back-to-top" /></a>
```

Comment:
@dkaiser014 Thanks. Here are the changes I want to suggest that I believe would best adapt to our codebase.
1. Instead of the [back icon](https://i.ibb.co/2PS8bhR/up-arrow.png) used by [best-of](https://github.com/best-of-lists/best-of/blob/main/README.md) I'd recommend my own slightly modified and optimized `png`, it's only 832b compared to 3.32kb.
![up_arrow](https://user-images.githubusercontent.com/21981508/106390105-69791600-63ac-11eb-9b0f-7997b70b43ba.png)
2. You can upload/save this new icon in `images/` directory with name `up_arrow.png`. So the new code now should look like:
```md
[<img align="right" width="16" height="16" src="images/up_arrow.png" alt="Back to top">](#contents)
```
Notice the height/width is now `16px`, and I wrapped it with markdown syntax rather than raw HTML. This format aligns with the rest of the markdown code within `README.md`. Appreciate your contribution on this.

Reply:
@mhxion Thanks for the pointers on how it should look! I'll get working on it asap